### PR TITLE
Feature/hero 1

### PR DESCRIPTION
**Pull request by saurabh-lohiya**

### CODE REVIEW

File: index.ftd [hunk 1]
```diff
@@ -1,40 +1,1130 @@
...
+spacing: space-between
+
+-- ftd.column:
+width.fixed.percent: 45
+width if { ftd.device == "mobile" }: fill-container
+spacing.fixed.px: 15
+align-self: center
+
+-- ftd.text: $hero-secondary.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:284]:
```suggestion
```

File: index.ftd [hunk 2]
```diff
@@ -1,40 +1,1130 @@
...
+width.fixed.percent: 50
+width if { ftd.device == "mobile" }: fill-container
+align-content: center
+align-self: center
+
+-- ftd.column:
+width.fixed.percent: 65
+
+-- ftd.text: $hero-tertiary.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:373]:
```suggestion
```

File: index.ftd [hunk 3]
```diff
@@ -1,40 +1,1130 @@
...
+wrap: true
+
+-- ftd.column:
+width.fixed.percent: 50
+width if { ftd.device == "mobile" }: fill-container
+align-content: center
+align-self: center
+
+-- ftd.column:
+width.fixed.percent: 65
```
Comment (gsalunke) [inline index.ftd:370]:
```suggestion
-- ftd.column:
width.fixed.percent: 65
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 4]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.text: $hero-tertiaryV2.title
+color: $inherited.colors.text-strong
+role: $inherited.types.heading-large
+style: bold
+margin-top.px: 15
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:491]:
```suggestion
```

File: index.ftd [hunk 5]
```diff
@@ -1,40 +1,1130 @@
...
+-- end: ftd.column
+
+-- ftd.column:
+width.fixed.percent: 50
+width if { ftd.device == "mobile" }: fill-container
+align-content: center
+align-self: center
+
+-- ftd.column:
+width.fixed.percent: 65
```
Comment (gsalunke) [inline index.ftd:482]:
```suggestion
-- ftd.column:
width.fixed.percent: 65
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 6]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.text: $hero-tertiary-with-search-bar.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:567]:
```suggestion
```

File: index.ftd [hunk 7]
```diff
@@ -1,40 +1,1130 @@
...
+role: $inherited.types.heading-large
+style: bold
+margin-top.px: 15
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:573]:
```suggestion
```

File: index.ftd [hunk 8]
```diff
@@ -1,40 +1,1130 @@
...
+wrap: true
+
+-- ftd.column:
+width.fixed.percent: 50
+width if { ftd.device == "mobile" }: fill-container
+align-content: center
+align-self: center
+
+-- ftd.column:
+width.fixed.percent: 65
```
Comment (gsalunke) [inline index.ftd:564]:
```suggestion
-- ftd.column:
width.fixed.percent: 65
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 9]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.text: $hero-tertiary-with-search-barV2.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:675]:
```suggestion
```

File: index.ftd [hunk 10]
```diff
@@ -1,40 +1,1130 @@
...
+role: $inherited.types.heading-large
+style: bold
+margin-top.px: 15
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:681]:
```suggestion
```

File: index.ftd [hunk 11]
```diff
@@ -1,40 +1,1130 @@
...
+-- end: ftd.column
+
+-- ftd.column:
+width.fixed.percent: 50
+width if { ftd.device == "mobile" }: fill-container
+align-content: center
+align-self: center
+
+-- ftd.column:
+width.fixed.percent: 65
```
Comment (gsalunke) [inline index.ftd:672]:
```suggestion
-- ftd.column:
width.fixed.percent: 65
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 12]
```diff
@@ -1,40 +1,1130 @@
...
+wrap: true
+spacing: space-between
+
+-- ftd.column:
+width.fixed.percent: 45
+width if { ftd.device == "mobile" }: fill-container
+align-self: center
+
+-- ftd.text: $hero-with-search-bar.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:753]:
```suggestion
```

File: index.ftd [hunk 13]
```diff
@@ -1,40 +1,1130 @@
...
+role: $inherited.types.heading-large
+style: bold
+margin-top.px: 15
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:759]:
```suggestion
```

File: index.ftd [hunk 14]
```diff
@@ -1,40 +1,1130 @@
...
+width: fill-container
+max-width.fixed.px: $common.max-width
+height: fill-container
+wrap: true
+spacing: space-between
+
+-- ftd.column:
+width.fixed.percent: 45
+width if { ftd.device == "mobile" }: fill-container
+align-self: center
```
Comment (gsalunke) [inline index.ftd:750]:
```suggestion
-- ftd.column:
width.fixed.percent: 45
width if { ftd.device == "mobile" }: fill-container
align-self: center
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 15]
```diff
@@ -1,40 +1,1130 @@
...
```
Comment (gsalunke) [inline index.ftd:842]:
```suggestion
-- ftd.column:
width.fixed.percent: 45
width if { ftd.device == "mobile" }: fill-container
align-self: center
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 16]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.text: $hero-with-search-barV2.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:845]:
```suggestion
```

File: index.ftd [hunk 17]
```diff
@@ -1,40 +1,1130 @@
...
+role: $inherited.types.heading-large
+style: bold
+margin-top.px: 15
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:851]:
```suggestion
```

File: index.ftd [hunk 18]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.text: $hero-cover.title
+color: $inherited.colors.text-strong
+role: $inherited.types.heading-large
+max-width.fixed.percent: 60
+text-align: center
+style: bold
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:1024]:
```suggestion
```

File: index.ftd [hunk 19]
```diff
@@ -1,40 +1,1130 @@
...
+align-content: center
+padding-vertical.px: 100
+
+-- ftd.column:
+width: fill-container
+max-width.fixed.px: $common.max-width 
+align-content: center
+
+-- ftd.text: $hero-cover.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:1017]:
```suggestion
```

File: index.ftd [hunk 20]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.column:
+width: fill-container
+align-content: center
+padding-vertical.px: 100
+
+-- ftd.column:
+width: fill-container
+max-width.fixed.px: $common.max-width 
+align-content: center
```
Comment (gsalunke) [inline index.ftd:1014]:
```suggestion
-- ftd.column:
width: fill-container
max-width.fixed.px: $common.max-width 
align-content: center
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 21]
```diff
@@ -1,40 +1,1130 @@
...
```
Comment (gsalunke) [inline index.ftd:1085]:
```suggestion
-- ftd.column:
width: fill-container
max-width.fixed.px: $common.max-width 
align-content: center
color: $inherited.colors.text-strong
```

File: index.ftd [hunk 22]
```diff
@@ -1,40 +1,1130 @@
...
+
+-- ftd.text: $hero-cover-with-search-bar.title
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:1088]:
```suggestion
```

File: index.ftd [hunk 23]
```diff
@@ -1,40 +1,1130 @@
...
+role: $inherited.types.heading-large
+max-width.fixed.percent: 60
+text-align: center
+style: bold
+
+-- ftd.text:
+color: $inherited.colors.text-strong
```
Comment (gsalunke) [inline index.ftd:1095]:
```suggestion
```

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: index.ftd (hunk 5 above)
Comment (saurabh-lohiya) [reply index.ftd:479]: I've resolved this But still it's showing not showing as outdated

File: index.ftd (hunk 8 above)
Comment (saurabh-lohiya) [reply index.ftd:560]: I've resolved this But still it's showing not showing as outdated

**Review by gsalunke — APPROVED** [review]

**Pull request merged**